tests/phpunit/tests/query/searchColumns.php

```diff
@@ -0,0 +1,381 @@
+<?php
+/**
+ * Testing the search columns support in `WP_Query`.
+ *
+ * @package WordPress\UnitTests
+ * @since 6.2.0
+ */
+
+/**
+ * Test cases for the search columns feature.
+ *
+ * @group query
+ * @group search
+ *
+ * @since 6.2.0
+ */
+class Tests_Query_SearchColumns extends WP_UnitTestCase {
+	/**
+	 * The post ID of the first fixture post.
+	 *
+	 * @since 6.2.0
+	 * @var int $pid1
+	 */
+	protected static $pid1;
+
+	/**
+	 * The post ID of the second fixture post.
+	 *
+	 * @since 6.2.0
+	 * @var int $pid2
+	 */
+	protected static $pid2;
+
+	/**
+	 * The post ID of the third fixture post.
+	 *
+	 * @since 6.2.0
+	 * @var int $pid3
+	 */
+	protected static $pid3;
+
+	/**
+	 * Create posts fixtures.
+	 *
+	 * @param WP_UnitTest_Factory $factory The factory instance.
+	 */
+	public static function wpSetUpBeforeClass( $factory ) {
+		self::$pid1 = self::factory()->post->create(
+			array(
+				'post_status'  => 'publish',
+				'post_title'   => 'foo title',
+				'post_excerpt' => 'foo excerpt',
+				'post_content' => 'foo content',
+			)
+		);
+		self::$pid2 = self::factory()->post->create(
+			array(
+				'post_status'  => 'publish',
+				'post_title'   => 'bar title',
+				'post_excerpt' => 'foo bar excerpt',
+				'post_content' => 'foo bar content',
+			)
+		);
+
+		self::$pid3 = self::factory()->post->create(
+			array(
+				'post_status'  => 'publish',
+				'post_title'   => 'baz title',
+				'post_excerpt' => 'baz bar excerpt',
+				'post_content' => 'baz bar foo content',
+			)
+		);
+	}
+
+	/**
+	 * The search should use default search columns when search columns are empty.
```

Comment:
```suggestion
	 * Tests that search uses default search columns when search columns are empty.
```
Test method descriptions should start with "Tests that...".

🔢 Applies to other test methods in the PR

Reply:
I've gone through and updated all the descriptions.